apps/web/components/Price/Price.vue

```diff
@@ -19,7 +19,11 @@ const { price } = props;
 
 const actualPrice = computed(() => {
   if (price && (price.special !== null || price.regular !== null)) {
-    return price.special ?? price.regular ?? 0;
+    let result = price.special ?? 0;
+    if (price.regular) {
+      result = price.regular;
+    }
+    return result;
```

Comment:
Are you sure this is correct ? If it exists, the regular price will always override the special one.